store/mockstore/mocktikv/rpc.go

```diff
@@ -248,8 +248,8 @@ func (h *rpcHandler) handleKvScan(req *kvrpcpb.ScanRequest) *kvrpcpb.ScanRespons
 	if !h.checkKeyInRegion(req.GetStartKey()) {
 		panic("KvScan: startKey not in region")
 	}
-	endKey := h.endKey
-	if len(req.EndKey) > 0 && (len(endKey) == 0 || bytes.Compare(req.EndKey, endKey) < 0) {
+	endKey := MvccKey(h.endKey).Raw()
```

Comment:
```suggestion
	endKey := h.rawEndKey
```

Reply:
@lonng , `h.rawEndKey` maybe doesn't set. And If alway set `h.rawEndKey`when set `h.endKey`, you will got some other error when `make test`.